client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,673 @@
+	'Music classes': i18n.translate( 'Music classes' ),
+	'Music lessons': i18n.translate( 'Music lessons' ),
+	'Music production': i18n.translate( 'Music production' ),
+	Musician: i18n.translate( 'Musician' ),
+	Mystery: i18n.translate( 'Mystery' ),
+	'Natural health products': i18n.translate( 'Natural health products' ),
+	'Natural health remedies': i18n.translate( 'Natural health remedies' ),
+	'Network marketing': i18n.translate( 'Network marketing' ),
+	Networking: i18n.translate( 'Networking' ),
+	News: i18n.translate( 'News' ),
+	Newspaper: i18n.translate( 'Newspaper' ),
+	'Non-profit': i18n.translate( 'Non-profit' ),
+	Novel: i18n.translate( 'Novel' ),
+	Nutrition: i18n.translate( 'Nutrition' ),
+	'Online business': i18n.translate( 'Online business' ),
+	'Online store': i18n.translate( 'Online store' ),
+	Optician: i18n.translate( 'Optician' ),
+	Optometrist: i18n.translate( 'Optometrist' ),
+	'Organic farming': i18n.translate( 'Organic farming' ),
+	Organization: i18n.translate( 'Organization' ),
+	Outdoors: i18n.translate( 'Outdoors' ),
+	Outfits: i18n.translate( 'Outfits' ),
+	Painter: i18n.translate( 'Painter' ),
+	Painting: i18n.translate( 'Painting' ),
+	Paranormal: i18n.translate( 'Paranormal' ),
+	Parenting: i18n.translate( 'Parenting' ),
+	'Parties and Festivities': i18n.translate( 'Parties and Festivities' ),
+	'Party promotion': i18n.translate( 'Party promotion' ),
+	'Performance art': i18n.translate( 'Performance art' ),
+	Performer: i18n.translate( 'Performer' ),
+	Perfume: i18n.translate( 'Perfume' ),
+	Personal: i18n.translate( 'Personal' ),
+	'Personal blog': i18n.translate( 'Personal blog' ),
+	'Personal development': i18n.translate( 'Personal development' ),
+	'Personal experiences': i18n.translate( 'Personal experiences' ),
+	'Personal finance': i18n.translate( 'Personal finance' ),
+	'Personal issues': i18n.translate( 'Personal issues' ),
+	'Personal trainer': i18n.translate( 'Personal trainer' ),
+	'Personal training': i18n.translate( 'Personal training' ),
+	'Personal website': i18n.translate( 'Personal website' ),
+	'Pet services': i18n.translate( 'Pet services' ),
+	Pets: i18n.translate( 'Pets' ),
+	Philosopher: i18n.translate( 'Philosopher' ),
+	Philosophy: i18n.translate( 'Philosophy' ),
+	Photocopier: i18n.translate( 'Photocopier' ),
+	'Photocopier repairs': i18n.translate( 'Photocopier repairs' ),
+	Photographer: i18n.translate( 'Photographer' ),
+	Photography: i18n.translate( 'Photography' ),
+	Photos: i18n.translate( 'Photos' ),
+	'Physical therapy': i18n.translate( 'Physical therapy' ),
+	Physiotherapy: i18n.translate( 'Physiotherapy' ),
+	'Pizza shop': i18n.translate( 'Pizza shop' ),
+	Plumber: i18n.translate( 'Plumber' ),
+	Podcast: i18n.translate( 'Podcast' ),
+	Poems: i18n.translate( 'Poems' ),
+	Poetry: i18n.translate( 'Poetry' ),
+	Pokémon: i18n.translate( 'Pokémon' ),
+	'Political activism': i18n.translate( 'Political activism' ),
+	'Political candidate': i18n.translate( 'Political candidate' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Policial candidate' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).